UI/MapWnd.cpp

```diff
@@ -1939,26 +1939,94 @@ void MapWnd::RenderSystems() {
             GG::Pt circle_ul = middle - circle_half_size;
             GG::Pt circle_lr = circle_ul + circle_size;
 
+            GG::Pt circle_distance = GG::Pt(GG::X(4), GG::Y(4));
+            
+            GG::Pt outer_circle_ul = circle_ul - circle_distance;
+            GG::Pt outer_circle_lr = circle_lr + circle_distance;
+
             if (fog_scanlines
                 && (universe.GetObjectVisibilityByEmpire(system_icon.first, empire_id) <= VIS_BASIC_VISIBILITY))
             {
                 m_scanline_shader.SetColor(GetOptionsDB().Get<GG::Clr>("UI.system-fog-of-war-clr"));
                 m_scanline_shader.RenderCircle(circle_ul, circle_lr);
             }
 
-            // render circles around systems that have at least one starlane, if circles are enabled.
+            // render circles around systems that have at least one starlane, if they are enabled
             if (circles) {
                 if (std::shared_ptr<const System> system = GetSystem(system_icon.first)) {
                     if (system->NumStarlanes() > 0) {
-                        glColor(GetOptionsDB().Get<GG::Clr>("UI.unowned-starlane-colour"));
 
+                        bool has_empire_planet = false;
```

Comment:
This variable isn't required.  `!colony_count_by_empire_id.empty()` fits the same purpose.